src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/PosBlockContextRule.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Threading.Tasks;
+
+namespace Stratis.Bitcoin.Features.Consensus.Rules.CommonRules
+{
+    /// <summary>
+    /// Context checks on a POS block.
+    /// </summary>
+    public class PosBlockContextRule : PosConsensusRule
```

Comment:
@Aprogiena so you are aware of this code (I guess its a vulnerability)

Reply:
Do you mean that block flood attack or something else?

Reply:
If so, then yes I have it my list

Reply:
I think it will be a good idea to also add this info to the rule class summaries

Reply:
Yes it will when we implement it.